### PR TITLE
Add an LB privacy policy

### PR DESCRIPTION
**Pull request by mayhem**
As per YT's bullshit policies, we're adding an LB specific privacy policy.

image

### CODE REVIEW

File: listenbrainz/webserver/templates/navbar.html [hunk 1]
```diff
@@ -58,6 +42,22 @@
             <li><a href="{{ url_for('index.similar_users') }}">Top similar users</a></li>
           </ul>
         </li>
+        <li class="dropdown">
```
Comment (mayhem) [inline listenbrainz/webserver/templates/navbar.html:45]: The about menu in the middle of the menu bar felt weird, thus I moved it to the end. Looks much better.

File: listenbrainz/webserver/templates/index/privacy-policy.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  <h3>Use of your YouTube profile information</h3>
+
+  <p>
+    The embedded music player that is present in a number of pages on ListenBrainz can
+    <a href="{{ url_for('profile.music_services_details') }}">optionally be connected to
+    play YouTube content</a>. During this connection process ListenBrainz will request
+    permissions to view your YouTube profile data. Please note that YouTube does not have
+    any permissions lesser than this permission; if we wish to play YouTube content on
+    ListenBrainz pages, we are required to ask for this permission, even though we wish we
+    didn't have to do this.
```
Comment (amCap1712) [inline listenbrainz/webserver/templates/index/privacy-policy.html:26]: We can probably improve the wording of these lines. maybe @reosarevok has some suggestions ?

**Review by amCap1712 — APPROVED** [review]
:100:

File: listenbrainz/webserver/templates/index/privacy-policy.html (hunk 2 above)
Comment (mayhem) [reply listenbrainz/webserver/templates/index/privacy-policy.html:26]: I dont see anything wrong with it -- what do you feel is amiss?

Comment (reosarevok) [reply listenbrainz/webserver/templates/index/privacy-policy.html:26]: "Please note that YouTube does not have any permission options that are less wide than this one" maybe?

Comment (MonkeyDo) [reply listenbrainz/webserver/templates/index/privacy-policy.html:26]: How about this?
"Please note that in order to search for YouTube content to play on ListenBrainz pages, we have no choice but to ask for this permission.
We only use the absolute minimum data required to play music through Youtube (authentication to your Youtube account), and do not use it for any other purpose."

Comment (atj) [reply listenbrainz/webserver/templates/index/privacy-policy.html:26]: > The embedded music player that is present in a number of pages on ListenBrainz can optionally be connected to    YouTube in order to play content. During the connection process ListenBrainz will request permissions to view your YouTube profile data. Unfortunately this is the minimum permission level offered by YouTube and thus if we wish to offer the ability to play YouTube content on ListenBrainz pages, we are required to ask for this permission, even though we do not need or want access to your YouTube profile. In summary, your data is used only to authenticate to your YouTube account and for no other purposes.

Comment (mayhem) [reply listenbrainz/webserver/templates/index/privacy-policy.html:26]: I'll accept you suggestion with a minor edit. Thanks @atj !

**Pull request merged**